src/services/ftp/backend.rs

```diff
@@ -218,7 +187,7 @@ impl Builder {
 pub struct Backend {
     endpoint: String,
     root: String,
-    client: Arc<Mutex<FtpStream>>,
+    credential: (String, String),
```

Comment:
I prefer to use `username` and `password` here.

Easier for reading and understanding.

Reply:
Done